addons/mail/static/src/core/common/channel_member_model.js

```diff
@@ -82,7 +82,7 @@ export class ChannelMember extends Record {
     typingTimeoutId;
 
     get name() {
-        return this.persona.nameOrDisplayName;
+        return this.persona.name;
```

Comment:
Seems like this getter is meaningless. It's only used once or twice, and now the field is not too long, so I would just suggest to use `member.persona.name` when needed

Reply:
I'd keep it, `member.name` is shorter than `member.persona.name`, for the amount being used this makes code nicer

Reply:
it's used only twice I think?

Reply:
The getter doesn't cost much, and it eases using `member` model.

The composition between persona and member is more about technical limitation, it sucks to read `member.persona.name` everywhere. Code could even be weak against lack of `persona`, this works only because we made sure to not lack fetching it. The getter could handle lack of persona in case there's a need.

I don't see a point to not remove a getter to save few LOCs and making using this model less practical.